src/plugins/intel_gpu/tests/unit/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -1680,6 +1681,76 @@ TEST(activation_i32_fw_gpu, basic_yxfb_i32_funcs) {
             case activation_func::floor:
                 ASSERT_EQ((int32_t)std::floor(input_ptr[i]), output_ptr[i]);
                 break;
+            case activation_func::abs:
+                ASSERT_EQ(std::abs(static_cast<int32_t>(input_ptr[i])), output_ptr[i]);
+                break;
+            default:
+                break;
+            }
+        }
+    }
+}
+
+TEST(activation_i32_fw_gpu, basic_yxfb_u8_funcs) {
+    auto& engine = get_test_engine();
+    auto input = engine.allocate_memory({ data_types::u8, format::yxfb,{ 2, 2, 2, 2 } });
+
+    std::vector<uint8_t> input_vec = {
+        1,   0,  5,   1,
+        2,   0,  6,  5,
+        3,   0, 7,  12,
+        4,   0, 8,   8
```

Comment:
to test abs, I think we need to have negative values too

Reply:
remove 'basic_yxfb_u8_funcs' tests